### ISSUE TITLE
Question to use

### ISSUE THREAD

**Issue by vallemar** [ontrackms/nativescript-dotenv#1]
WOW, I am very interested in this plugin, which I would have to put in the target files to replace by the environment values

**Comment by insytes** [comment]
Yes, feel free to use this plugin. It is compatible with NativeScript >= 8.0. We would recommend using the v1.0.0 tag for now as there may be some breaking changes soon.

`npm i -D ontrackms/nativescript-dotenv#v1.0.0`
